kitchen.recipes.yml

```diff
@@ -329,26 +329,28 @@ suites:
               cidr1
               cidr2
   - name: ephemeral_drives
-    #driver:
-    #  instance_type: m5d.xlarge  # instance type with ephemeral drives
+    driver:
+      instance_type: m5d.xlarge  # instance type with ephemeral drives
     run_list:
       - recipe[aws-parallelcluster::add_dependencies]
       - recipe[aws-parallelcluster-config::ephemeral_drives]
     verifier:
       controls:
         - ephemeral_drives_service
```

Comment:
Nitpick: should be a past participle, as the line below.

Reply:
changed, thanks